documentation/INSTALLATION.md

```diff
@@ -31,6 +31,8 @@ The following video shows the steps in action:
 
 [![How to install PersonalAnalytics on macOS](https://markdown-videos-api.jorgenkh.no/youtube/ovfRzp3Ksgk)](https://youtu.be/ovfRzp3Ksgk)
 
+> If you want to completely remove or reinstall PersonalAnalytics on macOS, please visit the [Troubleshooting](TROUBLESHOOTING.md) page and follow the steps under "macOS: Reinstalling PersonalAnalytics from Scratch".
```

Comment:
can you please directly link to the paragraph (and shorten the text instead)? (since i am expecting that the troubleshooting page will grow)